LanguageFeatures/Augmentation-libraries/augmented_expression_A01_t05.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The exact result of an `augmented` expression depends on what is
+/// being augmented, but it generally follows the same rules as any normal
+/// identifier:
+/// - Augmenting getters: Within an augmenting getter `augmented` invokes the
+///   getter and evaluates to the return value. If augmenting a field with a
+///   getter, this will invoke the implicit getter from the augmented field.
+///
+/// @description Checks that it is a compile-time error if within an augmenting
+/// getter `augmented +` expression is invoked.
```

Comment:
It's slightly dangerous to talk about the `augmented +` expression: `augmented` works in a way that is similar to `super` in the narrow context where we're augmenting the corresponding kind of declaration: `augmented + e` works similarly to `super + e` when it occurs in an augmentation of an `operator +` declaration (and that's just because `augmented` is not an expression, it's part of the special term `augmented + e` which is an invocation of the augmented `operator +` with the actual argument `e`). This does not happen when `augmented` occurs in an augmentation of a getter declaration, in that case `augmented + e` is a regular binary operator expression where `augmented` is a getter whose return type should have an `operator +`.

Reply:
Description is rewritten to avoid "`augmented +` expression"